docs/Installation.rst

```diff
@@ -1,3 +1,161 @@
+- `pdf_report` (bool): Specify whether pdf report of the simulation's results is generated or not (Command line "-pdf"). Default: False.
+- `save_png` (bool): Specify whether png figures with the simulation's results are generated or not (Command line "-png"). Default: False.
+
+
+Edit the csv files (or, for devs, the json file) and run the `main()` function. The following `kwargs` are possible:
+
+
+##### Default settings
+
+If you execute the `mvs_tool` command in a path where there is a folder named `inputs` (you can use the folder
+`input_template` for inspiration) this folder will be taken as default input folder and you can simply run
+
+    `mvs_tool`
+
+A default output folder will be created, if you run the same simulation several time you would
+ have to either overwrite the existing output file with
+
+    `mvs_tool -f`
+
+Or provide another output folder's path
+
+    `mvs_tool -o <path_to_other_output_folder>`
+
+## Generate pdf report or an app in your browser to visualise the results of the simulation
+
+To use the report feature you need to install extra dependencies first
+
+    `pip install multi-vector-simulator[report]`
+
+### Generate a report after running a simulation
+
+Use the option `-pdf` in the command line  `mvs_tool` to generate a pdf report in the
+ simulation's output folder (by default in `MVS_outputs/report/simulation_report.pdf`):
+
+    `mvs_tool -pdf`
+
+### Generate only the figures of a simulation's results
+
+Use the option `-png` in the command line  `mvs_tool` to generate png figures of the results in the
+ simulation's output folder (by default in `MVS_outputs/`):
+
+    `mvs_tool -png`
+
+
+### post-processing
+To generate a report of the simulation's results, run the following command **after** a simulation
+ generated an output folder:
+
+    `mvs_report -i path_simulation_output_folder -o path_pdf_report`
+
+where `path_simulation_output_folder` should link to the folder of your simulation's output, or directly to a json file (default `MVS_outputs/json_input_processed.json`)
+and `path_pdf_report` is the path where the report should be saved as a pdf file.
+
+
+The report should appear in your browser (at http://127.0.0.1:8050) as an interactive Plotly Dash app.
+
+You can then print the report via your browser print functionality (ctrl+p), however the layout of
+ the pdf report is only well optimized for chrome or chromimum browser.
+
+It is also possible to automatically save the report as pdf by using the option `-pdf`
+
+    `mvs_report -i path_simulation_output_folder -pdf`
+
+By default, it will save the report in a `report` folder within your simulation's output folder default (`MVS_outputs/report/`).
+See `mvs_report.py -h` for more information about possible options. The css and images used to make the report pretty should be located under
+`report/assets`.
+
+.. check that there is no extra information below here that should be displayed above
+
```

Comment:
I only left the code below so that we can compare with `.md` formatted text, the `.md` text contains the must up-to-date information

Reply:
@mahendrark you also need to compare the text in `installation.rst` with the one displayed below this. Is it identical, or is there something missing?